### PR TITLE
first iteration on gates jobs

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

**Comment by elopio** [comment]
this is missing the plugin in the plugin files, right?

**Comment by fgimenez** [comment]
@elopio yep, fixed

File: containers/jenkins-master/config/jobs/os-gate-beta-rc/config.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  <builders/>
+  <publishers/>
+  <buildWrappers/>
+  <icon/>
+  <dsl>parallel(
+	{ build("snappy-daily-rolling-openstack", channel: "beta", type: "Promotion beta - rc") },
+    { build("snappy-rollbackstress-rolling-openstack", channel: "beta", type: "Promotion beta - rc") }
+)
+// the upload job will probably be different for each gate
+build("github-snappy-daily-ppa")</dsl>
```
Comment (elopio) [inline containers/jenkins-master/config/jobs/os-gate-beta-rc/config.xml]: Yes, when beta passes we need to call a job that promotes it on the store. Snapcraft is getting the publish command, but that is manual for now.

File: containers/jenkins-master/config/jobs/os-gate-rc-stable/config.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  <builders/>
+  <publishers/>
+  <buildWrappers/>
+  <icon/>
+  <dsl>parallel(
+	{ build("snappy-daily-rolling-openstack", channel: "candidate", type: "Promotion rc - stable") },
+    { build("snappy-rollbackstress-rolling-openstack", channel: "candidate", type: "Promotion rc - stable") }
+)
+// the upload job will probably be different for each gate
+build("github-snappy-daily-ppa")</dsl>
```
Comment (elopio) [inline containers/jenkins-master/config/jobs/os-gate-rc-stable/config.xml]: I think this one should always be manually triggered. Even when we have the way to promote using snapcraft, somebody should trigger the job.

**Comment by elopio** [comment]
I'm +1, specially on adding the build-flow plugin. But I think we should remove the last trigger. The PPA is already uploaded by the time this things run.

**Comment by fgimenez** [comment]
@elopio ok, removed, thx

**Pull request merged**